integration_test_lite.py

```diff
@@ -145,16 +148,29 @@ async def get_bad_apis(self):
         print('Passing cases:')
         # Execute all tests in parallel
         results = await asyncio.gather(
-            *[self.bad_response(endpoint) for endpoint in endpoints]
+            *[self.test_endpoint(endpoint) for endpoint in endpoints]
         )
-        # Return all results that weren't None
-        return [result for result in results if result]
+
+        good_apis = [result for result in results if result['response_code'] == 200]
+        bad_apis = [result for result in results if result['response_code'] != 200]
```

Comment:
You actually iterate the entire `results` twice. I guess you can only loop just once.